### DISCUSSION TITLE
I'm getting pinia error after build

### DISCUSSION THREAD

**Discussion by GokhanKR** [quasarframework/quasar#16269] [CLI - SSR mode] [answered]
` yarn start
yarn run v1.22.19
$ node index.js
C:\Users\hng\Workspace\project\web\node_modules\pinia\dist\pinia.cjs:1700
            throw new Error(`[🍍]: "getActivePinia()" was called but there was no active Pinia. Did you forget to install pinia?\n` +
            ^

Error: [🍍]: "getActivePinia()" was called but there was no active Pinia. Did you forget to install pinia?
        const pinia = createPinia()
        app.use(pinia)
This will fail in production.
`
I've only used preFetch and a pinia store. My goal is to fetch data from a remote server using axios upon the initial load. I don't encounter any issues in development mode, but I'm facing this error when I build it.

**Comment by TobyMosque** [comment, marked as answer]
probably u're calling the store from outside of the setup.:
https://pinia.vuejs.org/core-concepts/outside-component-usage.html

That is warning, that can be or not a problem... if u're implement SSR, that is a problem, if not, probably not.

`preFetch`, `boots`, `navigation guards`, etc are executed outside of the setup, so u would pass the store if u wanna get ride of that warning:
```js
import { useMyStore } from 'src/stores/my'
defineOptions{{
  preFetch ({ store }) {
    const myStore = useMyStore(store)
  }
}}
```

**Comment by GokhanKR** [comment]
Yes, unfortunately it seems that way. I hadn't encountered any issues with Vuex before, but Pinia is causing problems. Now I'll need to reorganize and find a different solution :( Thanks.

**Reply by TobyMosque** [reply]
Just pass the store to the store definition as I did in my example.
